pom.xml

```diff
@@ -343,6 +343,11 @@
                                 <goals>
                                     <goal>report</goal>
                                 </goals>
+                                <configuration>
+                                    <title>${project.name} - ${scala.version}</title>
```

Comment:
I don't know where it will be displayed exactly, but won't Scala version be confused with the project version? E.g. `Commons - 2.12`?
Shouldn't it be rather something like `${project.name} [scala ${scala.version}]` or using Maven artifactId convention - `${project.name}_${scala.name}`?

Also shouldn't it be `${scala.binary.version}`?

Reply:
You are right. I was blind to that. Improved in commit 9a3482cd1e840d9a17646744c34831ca01165f80.

Reply:
so, `${scala.version}` or `${scala.binary.version}`? (2.12.15 or 2.12?)